charter-template.html

```diff
@@ -188,14 +188,21 @@ <h3>
           </p>
           <p class="todo">Consider making a second list for existing specifications that the group will maintain.</p>
           <dl>
-            <dt id="web-foo" class="spec"><a href="#">Web <i class="todo">[spec name]</i></a></dt>
+            <dt id="web-foo" class="spec"><a href="#"><i class="todo">[new spec name]</i></a></dt>
+            <dd>
+              <p>This specification will define <i class="todo">[concrete description]</i>.</p>
+
+              <p class="draft-status"><b>Draft state:</b> <i class="todo">[No draft | <a href="#">Use Cases and Requirements</a> | <a href="#">Editor's Draft</a> | <a href="#">Member Submission</a>]</i></p>
+              <p class="milestone"><b>Expected completion:</b> <i class="todo">[Q1–4 yyyy]</i></p>
+            </dd>
+            <dt id="web-foo" class="spec"><a href="#"><i class="todo">[current spec name]</i></a></dt>
             <dd>
               <p>This specification defines <i class="todo">[concrete description]</i>.</p>
 
-              <p class="draft-status"><b>Draft state:</b> <i class="todo">[No draft | <a href="#">Use Cases and Requirements</a> | <a href="#">Editor's Draft</a> | <a href="#">Member Submission</a> | <a href="#">Adopted from WG/CG Foo</a> | <a href="#">Working Draft</a>]</i></p>
+              <p class="draft-status"><b>Draft state:</b> <i class="todo">[<a href="#">Adopted from WG/CG Foo</a> | <a href="#">Working Draft</a>]</i></p>
               <p class="milestone"><b>Expected completion:</b> <i class="todo">[Q1–4 yyyy]</i></p>
               <p class="todo">Per <a href="https://www.w3.org/policies/process/#WGCharter">5.2.6</a>, for every Recommendation Track deliverable that continues work on a Working Draft (WD) published under any other Charter (including a predecessor group of the same name), for which there is an existing <a href='https://www.w3.org/policies/process/#exclusion-draft'>Exclusion Draft</a>: </p>
```

Comment:
@fantasai and @frivoal probably have stronger opinions, but I'd suggest something like

```html
<p class=todo>If this is a new technical report that will start as a copy of a different TR:
<p><b>Initial Text:</b> The <span class="todo">title, stable URL, and publication date of the existing technical report which will serve as the initial text of the deliverable.</span>
<p class=todo>Otherwise, if this group is taking over work on an existing technical report for which there is an existing <a href='https://www.w3.org/policies/process/#exclusion-draft'>Exclusion Draft</a>:
<p><b>Adopted Draft:....
```

Reply:
Sure. I tweaked further